app/hooks/stateModifiers.tsx

```diff
@@ -5,11 +5,25 @@ type StateString = string
 
 export const TOP = '-1'
 
+const uniqueChecker = (): ((e: StateEntry) => boolean) => {
```

Comment:
can we call it `makeUniqueChecker` or `uniqueCheckerFactory`?

Reply:
changed